test/framework/functional/framework.go

```diff
@@ -198,17 +200,19 @@ func (f *CollectorFunctionalFramework) DeployWithVisitors(visitors []runtime.Pod
 	}
 
 	log.V(2).Info("Generating Certificates")
+	certsDir := fmt.Sprintf("/tmp/clo-test-%d-%d", os.Getpid(), rand.Intn(10000)) //nolint:gosec
```

Comment:
This code looks duplicated.  Any reason we can not collect into a common package under test/framework

Reply:
@jcantrill Please take a look at the latest version